--- gtatool-2.0.1/src/conv-ffmpeg/media_object.cpp~	2018-04-29 18:57:24.000000000 +0200
+++ gtatool-2.0.1/src/conv-ffmpeg/media_object.cpp	2018-04-29 18:59:29.051110754 +0200
@@ -908,8 +908,6 @@
 #ifdef FF_API_LOWRES
             lowres = codec_ctx->lowres;
 #endif
-            if (lowres || (codec && (codec->capabilities & CODEC_CAP_DR1)))
-                codec_ctx->flags |= CODEC_FLAG_EMU_EDGE;
         }
         // Find and open the codec. AV_CODEC_ID_TEXT is a special case: it has no decoder since it is unencoded raw data.
         if (codec_ctx->codec_id != AV_CODEC_ID_TEXT && (!codec || (e = avcodec_open2(codec_ctx, codec, NULL)) < 0))
