--- gtatool-2.2.3/configure.ac~	2019-05-11 14:26:22.000000000 +0200
+++ gtatool-2.2.3/configure.ac	2019-05-11 14:29:11.905612327 +0200
@@ -460,10 +460,11 @@
     PKG_CHECK_MODULES([libpcl_io], [pcl_io >= 1.6], [],
         [PKG_CHECK_MODULES([libpcl_io], [pcl_io-1.7 >= 1.7], [],
             [PKG_CHECK_MODULES([libpcl_io], [pcl_io-1.8 >= 1.8], [],
+              [PKG_CHECK_MODULES([libpcl_io], [pcl_io-1.9 >= 1.9], [],
                 [pcd="no "
                 AC_MSG_WARN([PCL io library not found:])
                 AC_MSG_WARN([$libpcl_io_PKG_ERRORS])
-                AC_MSG_WARN([Disabled the from-pcd and to-pcd commands.])])])])
+                AC_MSG_WARN([Disabled the from-pcd and to-pcd commands.])])])])])
 fi
 AC_DEFINE_UNQUOTED([WITH_PCD], [`if test "$pcd" = "yes"; then echo "1"; else echo "0"; fi`], [Use pcd?])
 AM_CONDITIONAL([WITH_PCD], [test "$pcd" = "yes"])
