--- gtatool-2.0.1/src/conv-pfs/from-pfs.cpp~	2013-04-28 20:24:13.000000000 +0200
+++ gtatool-2.0.1/src/conv-pfs/from-pfs.cpp	2015-11-24 00:35:33.834608471 +0100
@@ -23,7 +23,7 @@
 
 #include <string>
 
-#include <pfs-1.2/pfs.h>
+#include <pfs/pfs.h>
 
 #include <gta/gta.hpp>
 
--- gtatool-2.0.1/src/conv-pfs/to-pfs.cpp~	2013-04-28 20:24:13.000000000 +0200
+++ gtatool-2.0.1/src/conv-pfs/to-pfs.cpp	2015-11-24 00:35:49.311263517 +0100
@@ -24,7 +24,7 @@
 #include <string>
 #include <limits>
 
-#include <pfs-1.2/pfs.h>
+#include <pfs/pfs.h>
 
 #include <gta/gta.hpp>
 
